src/chain/tezos/TezosNodeWriter.ts

```diff
@@ -74,13 +74,62 @@ export namespace TezosNodeWriter {
      *
      * @returns {string} Forged operation bytes (as a hex string)
      */
+    // TODO: move to an appropriate place
```

Comment:
Could we also log the operation payload that's being forged? It will be very useful for debugging.